src/docs/user/references/DataConnectors.adoc

```diff
@@ -7,127 +7,212 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
 
 ==== FTP and SFTP connectors
 
-*Task Variables:*
+*Variables:*
 The FTP and SFTP connectors share the same list of variables. Consequently, we describe them in the following table using a unique notation.
 `<PROTOCOL>` can take one of the following values: {`FTP`, `SFTP`}
 
-.FTP/SFTP_Connector_Task variables
-[cols="2,5,2"]
+.FTP/SFTP_Connector variables
```

Comment:
You don't need the _ (underscore) as it is not part of workflow/task names

Reply:
U r right.